README.md

```diff
@@ -197,7 +200,44 @@ Where:
     * `currentPage` is the retrieved page.
     * `isFirstPage` is a bool indicating if the page received is the first page
     * `isLastPage` is a bool indicating if the page received is the last page
-    
+
+## QR Codes
+This SDK offers the possibility to generate and consume transaction requests. Typically these actions should be done through the generation and scan of QR codes.
+
+### Generation
+To generate a new transaction request you can call:
+
+```kotlin
+val request = TransactionRequestParams(
+    type = TransactionRequestType.RECEIVE,
+    tokenId = "a_token_id",
+    amount = 10.24
```

Comment:
hmm we shouldn't have a float here, everything should be down to subunit to unit

Reply:
Say, if I want to receive 10.24`OMG` and token `OMG` has `subUnitToUnit` = `1,000`.

So I need to pass `amount` = `1024`, right?

Reply:
yup `10_240`